src/sandstorm/run-bundle.c++

```diff
@@ -1671,9 +1716,7 @@ private:
     return result;
   }
 
-  pid_t startNode(const Config& config) {
-    Subprocess process([&]() -> int {
-      // Create a listening socket for the meteor app on fd=3
+  void bindSocketToFd(const Config& config, uint port, uint targetFdNum) {
```

Comment:
The body of this method is over-indented... which admittedly helps make the diff clearer but should probably be fixed before merge.